Gruntfile.js

```diff
@@ -42,7 +42,8 @@ module.exports = function(grunt) {
           global_defs: defines,
           dead_code: false
         },
-        mangle: false
+        mangle: false,
+        report: 'gzip',
```

Comment:
Does the trailing comma cause any problems? I haven't yet learnt the JS syntax requirements here.